Yosemite/Yosemite/Actions/DataAction.swift

```diff
@@ -0,0 +1,15 @@
+import Foundation
+
+// MARK: - DataAction: Defines the actions that allows you to view all types of data available.
+//
+public enum DataAction: Action {
```

Comment:
Can we add clarification in a comment that it's about `/data/` APIs? Maybe say "WC data" with a link to API doc?
Otherwise this name and description look very generic.